src/app/components/challenge/challengeleaderboard/challengeleaderboard.component.scss

```diff
@@ -119,3 +119,23 @@
 	vertical-align: middle;
 	display: inline-block;
 }
+
```

Comment:
Is that CSS works perfectly for all the device size ?

Reply:
yes, Attaching another GIF for showing the same.